libs/components/core/src/index.ts

```diff
@@ -50,6 +50,7 @@ export { SkyHelpService } from './lib/modules/help/help.service';
 export { SkyIdModule } from './lib/modules/id/id.module';
 export { SkyIdService } from './lib/modules/id/id.service';
 
+export { SkyLayoutHostDirective } from './lib/modules/layout-host/layout-host.directive';
```

Comment:
Do we want to set a precedent of standalone directives, especially internal ones, without the lambda?

Reply:
We talked about this. Team is ok skipping the lambda.